methods.tex

```diff
@@ -43,11 +43,12 @@ \subsection*{Library organization}
 the ufunc will take care of looping over the dimensions of the inputs
 in such a way that if, say, \code{x} is a 1-D array with length 3, and
 \code{y} is a 2-D array with shape $2 \times 1$, the output will be
-an array with shape $2 \times 3$.  The ufunc machinery takes care
+an array with shape $2 \times 3$ (Fig.~\ref{fig:array-concepts}c).
+The ufunc machinery takes care
 of calling the function with all the appropriate combinations of
 input array elements to complete the output array.
 The elementary arithmetic operations of addition, multiplication, etc.,
-are implemented as ufuncs, so broadcasting applies even in expressions
+are implemented as ufuncs, so broadcasting applies also to expressions
```

Comment:
```suggestion
are implemented as ufuncs, so broadcasting applies even in expressions
```
stumbled over "even", but rereading "also" is too weak...

Reply:
How about "so that broadcasting applies also to"?

Reply:
Sounds good. The even does not really fit here I think, but it had the nice ring of being deeply ingrained into everything numpy does.